README.md

```diff
@@ -1 +1,39 @@
-# Kaidan
+# Kaidan - Cross plattform xmpp client
+
+Installation
+------------
+
+On Debian do the following:
+
+Create a working directory
+
+ * mkdir src
+ * cd src
+
+Fetch swift source
+
+ * wget https://github.com/swift/swift/archive/swift-3.0.zip
```

Comment:
Why not to download tarball from swift.im (https://swift.im/downloads/releases/swift-3.0/swift-3.0.tar.gz)? It's pretty the same but more logical. This way we get "swift-3.0" directory instead of "swift-swift-3.0"